docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+| Features               | RPC | Horizon | Galexie | Hubble |
+| ---------------------- | --- | ------- | ------- | ------ |
+| Real-time Data         | ✅  | ✅      | ✅      | ❌     |
+| Historical Data        | ❌  | ❌      | ✅      | ✅     |
+| Classic Data           | ✅  | ✅      | ✅      | ✅     |
+| Soroban Data           | ✅  | ❌      | ✅      | ✅     |
+| Transaction Submission | ✅  | ✅      | ❌      | ❌     |
+| Ad Hoc Data Analysis   | ❌  | ❌      | ❌      | ✅     |
+| Dashboarding           | ❌  | ❌      | ❌      | ✅     |
```

Comment:
I could use more ideas on what else we should put in the chart. I may have a bias towards Hubble lol

Reply:
another dimension could be curated/parsed data vs raw? 
I would collapse ad hoc data analysis and dashboarding into one dimension

Reply:
Yeah very fair. Removed dashboarding in a088c83120488cfcf7ef46830ab6621107828e2c